docs/achtergronddocumentatie/artefacten/0780.md

```diff
@@ -14,12 +14,13 @@ Deze oude issue kan vertaald worden naar de vraag of deze datums bij een klantco
 Nu zijn er in het klantcontavt een registratiedatum (vergelijkbaar met een creatiedatum) en een datum plaatsgevonden (vergelijkbaar met een indiendatum) opgenomen.
 
 Is het gewenst en nodig om de mutatiedatum op te nemen bij een klantcontact. 
-Het karakter van het klantdontact is dat het nietgewijzigd wordt. Er kan echter een reden zijn om één (of meerdere) correcties door te voeren. 
+Het karakter van het klantdontact is dat het niet gewijzigd wordt. Er kan echter een reden zijn om één (of meerdere) correcties door te voeren. 
 - Wat is het functionele belang van vastleggen van de mutatiedatum (of beter correctiedatum)?
 - Heb je het dan alleen over de laatste correctiedatum? 
 - Is de correctie zelf dan ook van belang?
 - Hier komen veel meer vragen rondom historie bij kijken
 
+Als een klantcontact vastgelegd is door een medewerker en (bv na het direct doorverbinden) daarna wordt aangepast door een 2e medewerker dan is dat weliswaar technsch een mutatie, maar functioneel betreft het één klantcontact. In dit geval is er geen noodzaak om een mutatiedatum vast te leggen.
```

Comment:
Is een 'mutatie' een 'correctie' - dat wordt hierboven genoemd als de 'correctere' term? Deze uitspraak betekent dat we het registreren van mutatiedatum niet in een bedrijfsregel kunnen afdwingen? Is dat handig/werkbaar?

Reply:
Het is nog steeds de vraag of je een mutatiedatum wil opnemen in een objecttype dat functioneel immutable is. Overigens is dit nog een vraagstuk dat we nog moeten bespreken. Daar komen we niet uit in een Pull Request denk ik.

Reply:
We nemen een correctiedatum op die bij het corrigeren van een een klantcontact verplicht gevuld wordt. We nemen tevens een datumtijd op waarmee we het overdrachtsmoment kunnen vastleggen bij een functionele aanvulling (technisch een wijziging) in geval van het direct doorverbinden van een klant aan een andere collega.